service/src/test/java/bio/terra/workspace/service/resource/referenced/ReferenceResourceServiceTest.java

```diff
@@ -228,6 +228,9 @@ void testDataRepoReference() {
           resultReferenceResource.castToDataRepoSnapshotResource();
       assertThat(resource, equalTo(resultResource));
 
+      referenceResourceService.validateReference(
```

Comment:
*referenced here and in filename

Reply:
`referenceResource` gets used in a lot of places (var names in this file, method names in ReferencedResourceService, etc). I filed PF-828 to fix this everywhere - doing this as a separate change to save this PR from too many unrelated changes